publisher/socials/telegram.py

```
@@ -0,0 +1,74 @@
+import os, copy
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)